Content.Server/Xenoarchaeology/Equipment/Components/TraversalDistorterComponent.cs

```diff
@@ -8,14 +8,14 @@
 public sealed partial class TraversalDistorterComponent : Component
 {
     [ViewVariables(VVAccess.ReadWrite)]
-    public BiasDirection BiasDirection = BiasDirection.In;
+    public BiasDirection BiasDirection = BiasDirection.Up;
 
     public TimeSpan NextActivation = default!;
     public TimeSpan ActivationDelay = TimeSpan.FromSeconds(1);
 }
 
 public enum BiasDirection : byte
 {
-    In, //down the tree, towards depth 0
-    Out //up the tree, away from depth 0
+    Up, //down the tree, towards depth 0
+    Down, //up the tree, away from depth 0
```

Comment:
`Up` is down the tree and `Down` is up the tree??

Reply:
Good catch, the comments are wrong :) Fixing!